Doc/library/functions.rst

```diff
@@ -892,6 +892,9 @@ are always available.  They are listed here in alphabetical order.
    sequence (such as a string, bytes, tuple, list, or range) or a collection
    (such as a dictionary, set, or frozen set).
 
+   **CPython implementation detail:**  ``len`` raises :class:`OverflowError`
+   on inputs larger than :data:`sys.maxsize`, such as :class:`range`.
```

Comment:
```suggestion
   on lengths larger than :data:`sys.maxsize`.
```

I don't understand the "such as range()" part. len(range(5)) returns 5, it doesn't fail with overflow?

Reply:
I've clarified that this is for cases such as `range(2 ** 64)`.